build/repo.props

```diff
@@ -1,10 +1,5 @@
 <Project>
   <ItemGroup>
-    <ExcludeFromTest Include="$(RepositoryRoot)test\Microsoft.AspNetCore.SignalR.Test.Server\*.csproj" />
```

Comment:
why is this no longer needed? Are tests project identified differently now?

Reply:
These project files no longer exist in these locations. Because they moved, this exclusion isn't necessary.